Eastern-Research-Group/csb-rebate-forms-app
# Feature/update bap bus query

## Related Issues:
* CSBAPP-5

## Main Changes:
Update BAP query for close out form data in include 'Old_Bus_Exclude__c' field and add 'oldBusExclude' field to a brand new close-out form submission

## Steps To Test:
1. Work with BAP team to get at least one Payment Request form submission into an "Accepted" state, so you can create a new Close-Out form submission. Ensure an excluded bus from a Payment Request form submission is showing as excluded in the Close-Out form.
